linux_os/guide/system/software/disk_partitioning/partition_for_var_log/rule.yml

```diff
@@ -42,7 +67,12 @@ references:
 
 platform: machine
 
+# (jhrozek): at the moment, the mount probe checks the /proc filesystem
+# even if openscap looks at a chroot, which doesn't allow to check for
+# mount points when checking RHCOS4 via CO
```

Comment:
So...right now this is effectively just a MANUAL check, right? Let's add a simple e2e test that merely checks that this results `MANUAL`. This way we'll remember to add a relevant test once the probe works.

Reply:
Done. I asked @evgenyz on the chat if the probe really can't work in chroot, but looking at openscap code, I think it can't, it really pokes at /proc and checks the mountpoints with `getmntent`, that won't work in a chroot.

In the end I don't think that a manual check matters here that much given that creating the mount points can only be done when the cluster is installed.

Reply:
@jhrozek well, the manual check will appear in the kube results and int he guide if they download the RAW ones. So it's relevant information that folks need to take into account.